src/screens/AmbientLightScreen.tsx

```
@@ -0,0 +1,84 @@
+        step = 0.01,
+        style,
+        ...rest
+      }: any) => (
+        <input
+          max={maximumValue}
+          min={minimumValue}
+          step={step}
+          style={StyleSheet.flatten(style)}
+          type="range"
+          value={value}
+          onChange={(e) => onValueChange(parseFloat(e.target.value))}
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)